tests/models/llama/test_modeling_llama.py

```diff
@@ -96,7 +98,8 @@ def prepare_config_and_inputs(self):
 
         input_mask = None
         if self.use_input_mask:
-            input_mask = random_attention_mask([self.batch_size, self.seq_length])
+            input_mask = torch.tril(torch.ones(7, 7))
```

Comment:
What about just:
```python
            input_mask = torch.tril(torch.ones(self.batch_size, self.seq_length))
```

=> this would work no?

Reply:
Yes, I forgot `tril` is defined for non-square tensors 😓